dandi/pynwb_utils.py

```diff
@@ -391,9 +393,23 @@ def validate(
     else:
         if version is not None:
             # Explicitly sanitize so we collect warnings.
-            # TODO: later cast into proper ERRORs
-            # Do we really need this custom internal function? string comparison works fine..
-            version = _sanitize_nwb_version(version, log=errors.append)
+            nwb_errors: list[str] = []
```

Comment:
Add `from __future__ import annotations` to the top of the file so that this will work.

Reply:
done, thanks